Makefile

```diff
@@ -0,0 +1,215 @@
+ifeq ($(shell uname),Darwin)
+SED := gsed
+else
+SED := sed
+endif
+
+CLEAN_TARGETS += protoc-clean prototool-clean
+GO_EXTRA_DEPS := install-protoc protoc-binaries install-mocker install-hub install-prototool install-protoc-gen-validate
+TEST_TARGETS += mods-vet
+GO_TEST_TARGET := lint-go run-go-tests
+BUILD_GO_OVERRIDE := generate-mocks compile
+GO111MODULE := on
+GO_USE_VENDOR ?= -mod=vendor
+export GO111MODULE
+
+include ./mk-include/cc-begin.mk
```

Comment:
Looks like these haven't been added. They're not typically added to public repositories in the first place.